.github/workflows/CI.yml

```
@@ -21,7 +21,6 @@ jobs:
           - '1'
           # - 'nightly'
         os:
-          - ubuntu-latest
           - windows-latest
```

To enable GitHub CI on linux for CPU

```suggestion
          - ubuntu-latest
          - windows-latest
```